proto/attestation/reference_value.proto

```diff
@@ -139,17 +153,15 @@ message KernelLayerReferenceValues {
   // Verifies the kernel based on endorsement.
   KernelBinaryReferenceValue kernel = 1;
 
-  // No longer used. Will be removed.
-  // TODO: b/325979696 - Validate the kernel command-line using a regex.
-  BinaryReferenceValue kernel_cmd_line = 2 [deprecated = true];
-
-  // Validates the kernel command-line using a regex.
-  RegexReferenceValue kernel_cmd_line_regex = 8;
+  // Validates the kernel command-line using either a string literal or a regex.
+  TextReferenceValue kernel_cmd_line_text = 9;
 
   // Fields are deprecated and kept only for backwards compatibility.
   // Remove ASAP.
-  BinaryReferenceValue kernel_setup_data = 3 [deprecated = true];
-  BinaryReferenceValue kernel_image = 7 [deprecated = true];
+  optional BinaryReferenceValue kernel_setup_data = 3 [deprecated = true];
+  optional BinaryReferenceValue kernel_image = 7 [deprecated = true];
+  RegexReferenceValue kernel_cmd_line_regex = 8 [deprecated = true];
+  optional BinaryReferenceValue kernel_cmd_line = 2 [deprecated = true];
```

Comment:
Why did they become optional though? I think they should be kept identical to what they were before. I mean, it's probably fine, but I would avoid unnecessary changes if possible. Also, being message types, I don't think there is an actual difference on the wire -- you can already set them to None, even without the optional, or did I misunderstand what optional actually does in proto3?

Reply:
Oops. an artifact from playing around with optional fields. Reverted.